README.md

```diff
@@ -0,0 +1,65 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Say Hello Microverse
```

Comment:
> You have done great to use the recommended readme template. 👍🏽 

- [ ] Now let use personalize this readme to your full description and ownership. 
- Add a short description message about the project features you implemented.
- Add clear instructions on how to get started including the necessary commands for the terminal.
- Under the getting started section, remove some sub-titles you have not used or implemented such as the `tests`, `deployment`, and others.